blockchain/hardhat.config.ts

```diff
@@ -3,17 +3,28 @@ import '@nomicfoundation/hardhat-toolbox';
 import 'hardhat-multibaas-plugin';
 import path from 'path';
 
-let deployerPrivateKey = '0x0000000000000000000000000000000000000000000000000000000000000000';
-let deploymentEndpoint, ethChainID, web3Key, adminApiKey, rpcUrl = '';
+let deployerPrivateKey = '0x0000000000000000000000000000000000000000000000000000000000000000',
+  deploymentEndpoint = '',
+  ethChainID: number,
+  web3Key,
+  adminApiKey: '',
+  rpcUrl = '';
```

Comment:
nit:

```suggestion
let deployerPrivateKey: string;
let deploymentEndpoint: string;
let ethChainID: number;
let web3Key: string;
let adminApiKey: string;
let rpcUrl: string;
```

If we don’t need to set any default values, we can omit them. Or set them to placeholders, such as, `<MULTIBAAS API KEY>`

IIRC, private key does need `0x`. Here is the private key `4244e4a26d283727d71b02a ... 92f708f4bea6121e3fbff2f` from MetaMask

What do you think?

Reply:
Indeed, this style is cleaner, but can you please confirm that it works with the linter? It may be easier to explain if we hop on a call, but basically there is a reason why there is a default `adminApiKey` value, but there is not one for `web3Key` for example.

Since web3 access may be provided through a MultiBaas reverse proxy, or an RPC url directly, there is some nuance to what values in the config are optional, and what values are required.

Reply:
> Indeed, this style is cleaner, but can you please confirm that it works with the linter?

Confirmed there are no issues when running linter

Reply:
> there is some nuance to what values in the config are optional, and what values are required.

I think leaving some comments on it to make it straightforward. What do you think?